htlcswitch/link.go

```diff
@@ -2074,7 +2064,7 @@ func (l *channelLink) processRemoteAdds(fwdPkg *channeldb.FwdPkg,
 			continue
 		}
 
-		heightNow := l.bestHeight
+		heightNow := l.cfg.Switch.BestHeight()
```

Comment:
the `l.bestHeight` can be removed after this change.

Reply:
Fixed.